packages/terra-alert/docs/MoreInformation.md

```diff
@@ -0,0 +1,166 @@
+# Terra Alert - More Information
+
+## Layout
+
+An Alert consists of the following elements:
+- Status bar: a colored border on the left of the banner where the color of the border corresponds to the type of the alert. It is one of the elements that helps the user to understand the severity of the notification.  For a custom alert, you can provide the color to be used for the status bar.
+- Icon: for each standard notification type, a specific icon is rendered to the left of the banner. The icon is another element that helps the user understand the severity and meaning of the notification.  For a custom alert, you can provide your own icon to be rendered in the banner.
+- Title: a brief word or phrase that summarizes the notification.  It is rendered in a bold font to gain the attention of the user.  For the standard notification types, if you don't provide a title, then pre-defined text is used.  For a custom alert, the title is optional.
+- Message content: the notification message which can consist of text or HTML.
+- Action area: an optional area to the right of the banner that contains actionable elements (typically buttons) allowing the user to respond to the alert notiifcation.  This area can consist of the following:
+  - Dismiss button: the Alert component can be configured to be dismissible.  If it is to be dismissible, a Dismiss button is rendered in the action area.
+  - Action element: you can configure the Alert component to include additional actionable element(s) in the action area.  typically you would include a button element.  To be consistent with the Dismiss button if it is used, you should use the Terra Button component.
+
+This image shows the layout of the elements of the Alert component.
+![Image](images/alertLayout.png?raw=true)
+
+## Props
```

Comment:
I'd be happy to remove it if its not appropriate here.  My reasoning for including it was that I wasn't sure how someone coming from NPM to the documentation for a component would get that information short of looking at the component source.  I know the props table is being generated in the site application, but  I didn't think an NPM user would be able to access that.  I may be missing some knowledge on how the components are to be documented, so I'll do whatever is agreed upon.